src/Data/Permissions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace App\Data;
+
+use App\Data\Entity\User;
+
+class Permissions {
+
+    /**
+     * @var Authenticator
+     */
+    protected $auth;
+
+    /**
+     * @var User
+     */
+    protected $user;
+
+    public function __construct(Authenticator $auth)
+    {
+        $this->auth = $auth;
+    }
+
+    public function setUser(User $user)
```

Comment:
I can do that. The reason I did it this way is because it was easier to handle with the DI. If I add it to the constructor, I'll have to share the currently logged in user to get it injected. Or manually construct it from within `Authenticator` or somewhere else.